spec/features/accession_web_archive_spec.rb

```diff
@@ -88,5 +89,12 @@
     expect(page).to have_link('thumbnail.jp2')
     expect(page).to have_content('image/jp2')
     expect(page).to have_content('400 px')
+
+    # Verify that the purl XML includes the proper archived website URL
+    purl_xml = "#{Settings.purl_url}/#{seed_druid}.xml"
```

Comment:
there's a way we get the purl_xml from dor-services-app so it's faster ... is this doing that?  See https://github.com/sul-dlss/infrastructure-integration-test/blob/main/spec/support/public_xml_helpers.rb 

I'm surprised this doesn't time out, tbh.  maybe because it's the purl xml, and not the purl??

Reply:
Hmm.. I'm happy to make that change.